documentation/site/content/index.html

```diff
@@ -21,7 +21,8 @@ <h2 class="govuk-heading-m">How the new Register early career teachers service w
   </li>
   <li>
     <a class="govuk-link--no-visited-state" href="/service-rules/lead-provider-users/">Lead provider users</a>
-    (<a class="govuk-link--no-visited-state" href="/service-rules/ecf1/lead-provider-users/"><span class="govuk-visually-hidden">Lead provider users in </span>ECF1</a>)
+    (<a class="govuk-link--no-visited-state" href="/service-rules/ecf1/lead-provider-users/"><span class="govuk-visually-hidden">Lead provider users in </span>ECF1</a>,
+    <a class="govuk-link--no-visited-state" href="/service-rules/ecf1/lead-provider-api/"><span class="govuk-visually-hidden">Lead provider </span>API</a>)
```

Comment:
do we want this to be Lead provider API in ECF1 instead?

Reply:
I think it's fine as is, keeps the bracketed text short.